docs/UserGuide.md

```diff
@@ -202,18 +202,35 @@ _Details coming soon ..._
-
+2. **Restrictions on the valid user input of `Faculty` values**: For the current version, a valid user for `Faculty` with prefix `f/` can only match exactly the same string as suggested below. Cases can be ignored though.
+   - Arts and Social Sciences
+   - Business
+   - Computing
+   - Continuing and Lifelong Education
+   - Dentistry
+   - Design and Environment
+   - Duke-NUS Medical School
+   - Engineering
+   - Integrative Sciences and Engineering
+   - Law
+   - Medicine
+   - Music
+   - Public Health
+   - Public Policy
+   - Science
+   - University Scholars Programme
+   - Yale-NUS College
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL v/VENUE m/MODULE [t/TAG]…​ [a/AVAILABILITY]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com v/123, Clementi Rd, 1234665 m/CS2103 t/friend t/colleague a/monday`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [v/VENUE] [m/MODULE] [t/TAG]…​ [a/AVAILABILITY]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**Sort** | `sort [ATTRIBUTE]`<br> e.g., `sort p/`
-**List** | `list`
-**Help** | `help`
+| Action     | Format, Examples                                                                                                                                                                                                                        |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL f/FACULTY v/VENUE m/MODULE [t/TAG]…​ [a/AVAILABILITY]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com f/Computing v/123, Clementi Rd, 1234665 m/CS2103 t/friend t/colleague a/monday` |
+| **Clear**  | `clear`                                                                                                                                                                                                                                 |
+| **Delete** | `delete INDEX`<br> e.g., `delete 3`                                                                                                                                                                                                     |
+| **Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [v/VENUE] [m/MODULE] [t/TAG]…​ [a/AVAILABILITY]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                                                                                 |
```

Comment:
missed out on adding faculty to edit command

Reply:
OK fixed now